### DISCUSSION TITLE
Right sidebar

### DISCUSSION THREAD

**Discussion by DutchPete** [notaapp/talk#417] [Questions]
The right sidebar consists of 2 'sub bars': one showing the workspace, the other one showing the files in the workspace.

AFAIK, the right sidebar can only be hidden in its entirety: is there a way to only hide the workspace sb-bar?

**Comment by astoilkov** [comment]
I think `Ctrl+2` does what you are thinking of:
![CleanShot 2024-01-03 at 15 00 40@2x](https://github.com/notaapp/talk/assets/884810/9eb0fc0d-5dcb-48a5-a85f-f42c07de04c3)

**Comment by DutchPete** [comment]
Got it! Thanks for your help.
